system/mavros2/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+
+version: '3'
+
+services:
+  simhost: # Must be called simhost
+    image: uobflightlabstarling/starling-sim-iris:latest
+    ports:
+      - "8080:8080"
+
+  sitl:
+    image: uobflightlabstarling/starling-sim-px4-sitl:nightly
```

Comment:
Do we want nightly here?

Reply:
I should probably get rid of this docker-compose, I was using it for testing actually